mods/d2k/rules/defaults.yaml

```diff
@@ -367,7 +367,6 @@
 	Selectable:
 		Priority: 2
 	RevealsShroud:
-		VisibilityType: CenterPosition
```

Comment:
Related to #15231, which is not fixed here.  This PR just removes the bogus definition.